packages/model-viewer/src/features/ar.ts

```diff
@@ -71,7 +72,7 @@ export declare interface ARInterface {
   arModes: string;
   arScale: string;
   iosSrc: string|null;
-  readonly canActivateAR: boolean;
+  xrEnvironment: boolean, readonly canActivateAR: boolean;
```

Comment:
I think you're looking for a `;` here instead of a `,`

Reply:
Updated, thanks!